ehrapy/preprocessing/_imputation.py

```diff
@@ -155,38 +141,30 @@ def simple_impute(
 
     _warn_imputation_threshold(adata, var_names, threshold=warning_threshold)
 
-    with Progress(
-        "[progress.description]{task.description}",
-        SpinnerColumn(),
-        refresh_per_second=1500,
-    ) as progress:
-        progress.add_task(f"[blue]Running simple imputation with {strategy}", total=1)
-        # Imputation using median and mean strategy works with numerical data only
-        if strategy in {"median", "mean"}:
-            try:
-                _simple_impute(adata, var_names, strategy)
-            except ValueError:
-                raise ValueError(
-                    f"Can only impute numerical data using {strategy} strategy. Try to restrict imputation"
-                    "to certain columns using var_names parameter or use a different mode."
-                ) from None
-        # most_frequent imputation works with non-numerical data as well
-        elif strategy == "most_frequent":
+    if strategy in {"median", "mean"}:
+        try:
             _simple_impute(adata, var_names, strategy)
-        # unknown simple imputation strategy
-        else:
-            raise ValueError(  # pragma: no cover
-                f"Unknown impute strategy {strategy} for simple Imputation. Choose any of mean, median or most_frequent."
+        except ValueError:
+            raise ValueError(
+                f"Can only impute numerical data using {strategy} strategy. Try to restrict imputation"
+                "to certain columns using var_names parameter or use a different mode."
             ) from None
+    # most_frequent imputation works with non-numerical data as well
+    elif strategy == "most_frequent":
+        _simple_impute(adata, var_names, strategy)
+    # unknown simple imputation strategy
+    else:
+        raise ValueError(  # pragma: no cover
```

Comment:
This could also be tested instead of a `pragma: no cover` but it's a trivial test.